### PR TITLE
Complete Task 5

### PR DESCRIPTION
**Pull request by waiwong-nc**

### CODE REVIEW

**Comment by waiwong-nc** [comment]
Please view the list of articles via path "/articles"

File: src/router.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import {
+  createBrowserRouter,
+  createRoutesFromElements,
+  Route
+} from "react-router-dom";
+
+import RootLayout from "./components/layout/RootLayout";
+import FrontPage from "./components/frontPage/FrontPage";
+import ArticlesPage from "./components/articles/ArticlesPage";
+import ArticlePage from "./components/articles/article/ArticlePage";
```
Comment (MRombi) [inline src/router.js:10]: The path is "./components/articles/article/articlePage"

File: src/components/articles/article/articlePage.js [hunk 2]
```diff
@@ -1,12 +1,73 @@
...
-      </div>
-    );
+      <ArticleFrame {...article} />
+    )
+  }
+
+  return (
+    <div className="article_page_container">
+      <BackwardButton url="/articles" _class="article_back_btn" />
+      {displayArticles()}
```
Comment (MRombi) [inline src/components/articles/article/articlePage.js:68]: Interesting use of displayArticles 😄

**Review by MRombi — APPROVED** [review]
Well done on this ticket, you are free to merge but please add some navigation on the home page! 😄

**Pull request merged**